Admin list

kelvinempires opened the pull request:
admin user list implemented

Commit:
Create navigation bar for landing page

Commit:
Replace 'T3 stack' contents with 'OpenTask'

onelove-dev committed:
Merge pull request #38 from Bhabishworgrg/feature/nav-bar

Create navigation bar for landing page

ekumamatthew committed:
dashboard layout

ekumamatthew committed:
dashboard layout

ekumamatthew committed:
Merge pull request #107 from manassehO/dashboard_layout

Dashboard layout

devJaja committed:
feat: Create “The Perks You’ll Experience” Section

Clement-coder committed:
Implement Modules section based on Figma design

truthixify committed:
feat: added erc20 interface and simple escrow contract

devJaja committed:
Create NavBar and Make It Responsive

priyanshu-samal committed:
added footer of landing page

favourwright committed:
feat: add GreetingCard component that dynamically displays greetings based on the time of day

ikemHood committed:
feat: init contracts and added types

onelove-dev committed:
Merge pull request #119 from manassehO/cairo-init

feat: Initialize OpenTask Cairo contract structure with core types and interfaces

truthixify committed:
resolved conflict and made changes to escrow contract

truthixify committed:
added erc20 interface

blurbeast committed:
reaward issue created

blurbeast committed:
rewardpaid event completed

blurbeast committed:
dependency resolved

ekumamatthew committed:
Merge pull request #118 from favourwright/feature/create-welcome-cards

feat: add GreetingCard component that dynamically displays greetings based on the time of day

ekumamatthew committed:
Merge pull request #117 from priyanshu-samal/footer-of-the-landing-page

added footer of landing page

ekumamatthew committed:
Merge pull request #116 from devJaja/feat/responsiveNavbar

Feat/responsive navbar

ekumamatthew committed:
Merge pull request #111 from Clement-coder/develop

Implement Modules section based on Figma design

ekumamatthew committed:
landing fix

ekumamatthew committed:
merge fix

hakymulla committed:
Implement fund_task Function

drakkomaximo committed:
Ui has been created

hakymulla committed:
remove interfaces

ekumamatthew committed:
landing page

ikemHood committed:
Merge pull request #120 from blurbeast/is87

Is87 || RewardPaid Event

blurbeast committed:
Is93 || Dispute events (#121)

* reaward issue created

* rewardpaid event completed

* dependency resolved

* dispute event added

* updated the lib file

* updated the events package

* dispute events done

ekumamatthew committed:
Merge pull request #125 from manassehO/landing_ui

Landing UI

ekumamatthew committed:
ui fix

ekumamatthew committed:
added left marging

ekumamatthew committed:
Merge pull request #127 from manassehO/fix/ui

Fix/UI

favourwright committed:
feat: Create and Validate OTP Input

hakymulla committed:
Merge branch 'develop' into fundtask

raymondjoseph02 committed:
feature: implement create password form input

ekumamatthew committed:
Merge pull request #128 from favourwright/feat/create-and-validate-OTP-input

feat: Create and Validate OTP Input

raymondjoseph02 committed:
fix: add max width

ekumamatthew committed:
Merge pull request #130 from raymondjoseph02/feature/implement_create_password_form

feature: implement create password form input

ekumamatthew committed:
auth_layout

ekumamatthew committed:
Merge pull request #132 from manassehO/auth_layout_screens

auth_layout

onelove-dev committed:
Merge pull request #115 from truthixify/fix-80

feat: added erc20 interface and simple escrow contract

hakymulla committed:
Merge branch 'develop' into fundtask

hakymulla committed:
used transfer_from_tokens function in interfaces/Ierc20

hakymulla committed:
fix import

onelove-dev committed:
Merge pull request #122 from hakymulla/fundtask

feat: Implement fund_task Function

dDevAhmed committed:
Email Validation (#133)

victocoder committed:
Feature/dashboard home Fixes #134  (#138)

* implemented dashboard home page with summary cards and recommended tasks

* implemented dashboard right side bar

* updated the right bar adding adding paragraph below the title

wheval committed:
add tasks page (#139)

* add tasks page

* fix: tasks page to

* fix: remove duplicate card-image

* add individul task page

---------

Co-authored-by: Ikem <91676649+ikemHood@users.noreply.github.com>

davedumto committed:
feat: build survey page ui (#140)

Co-authored-by: Ikem <91676649+ikemHood@users.noreply.github.com>

ekumamatthew committed:
fix dashboard layout

ikemHood committed:
added backend packages

ikemHood committed:
added backend packages

0xNgoo committed:
feat: integrate Better-Auth for user authentication (#169)

0xNgoo committed:
feat: integrate Better-Auth for user authentication (#169)

ikemHood committed:
Add GitHub workflow for UI builds and implement pre-commit linting

ikemHood committed:
Add GitHub workflow for UI builds and implement pre-commit linting

ikemHood committed:
Fix build errors by excluding contracts directory from TypeScript compilation

ikemHood committed:
Fix build errors by excluding contracts directory from TypeScript compilation

onelove-dev committed:
Merge pull request #171 from Vynix-Labs/ui-workflow

Add GitHub workflow for UI builds and implement pre-commit linting

ekumamatthew committed:
refactor: update sidebar routes and remove unused dashboard files

- Changed sidebar routes from "/dashboard/task" to "/task" and "/dashboard" to "/home".
- Deleted unused layout, page, and task detail files from the dashboard section.

ekumamatthew committed:
refactor: update sidebar routes and remove unused dashboard files

- Changed sidebar routes from "/dashboard/task" to "/task" and "/dashboard" to "/home".
- Deleted unused layout, page, and task detail files from the dashboard section.

ekumamatthew committed:
Merge branch 'fix_tableComponent' of https://github.com/Vynix-Labs/OpenTask into fix_tableComponent

ekumamatthew committed:
Merge pull request #172 from Vynix-Labs/fix_tableComponent

refactor: update sidebar routes and remove unused dashboard files

best2025j committed:
task web page done

best2025j committed:
changes made to the responsiveness

best2025j committed:
dashboard navbar error foxed

Royal-design committed:
fix task page

best2025j committed:
changes made

best2025j committed:
env error fix and merge fix also

best2025j committed:
merge issues fiex

Ajibose committed:
feat: Implemeted user and wallet schema, with otp generation and verification endpoints (#44) (#175)

* Implemented otp generation and verification

* Implemeted a better secure otp generation

* Put better comments

* Made OTP storage more secure by storing hashed format

ekumamatthew committed:
fix for dashboard layout

ekumamatthew committed:
fix for dashboard layout

raymondjoseph02 committed:
feature:ui for the admin disputes page

raymondjoseph02 committed:
feature:ui for the admin disputes page

ekumamatthew committed:
Merge pull request #174 from Royal-design/task-details-page

fix task page

best2025j committed:
faq added

best2025j committed:
Fix husky and ESLint empty object issues

best2025j committed:
table added to faq

best2025j committed:
slected filter is now working

best2025j committed:
guidelines added

best2025j committed:
button spaced for both edit and delect

raymondjoseph02 committed:
fix:text size on mobile

best2025j committed:
creator task page updated was mistaking deleted

kelvinempires committed:
first commit

raymondjoseph02 committed:
feature: implement the tasks ui page

best2025j committed:
changes made to faq page

best2025j committed:
changes made to faq ui

best2025j committed:
Fix workflow: remove conflicting paths and use paths-ignore

kelvinempires committed:
start working on admin list view

kelvinempires committed:
feat: add UserDetailPage component for displaying user information and task history

best2025j committed:
formatting error fixed

best2025j committed:
formatting error fixed

ekumamatthew committed:
Merge pull request #173 from best2025j/feat/design

 task web page done

raymondjoseph02 committed:
Merge branch 'develop' into feature/implement_dispute_ui_page

ekumamatthew committed:
Merge pull request #178 from raymondjoseph02/feature/implement_dispute_ui_page

Feature/implement dispute UI page

kelvinempires committed:
first commit

kelvinempires committed:
firts commit

kelvinempires committed:
user list structure

kelvinempires committed:
user details page implimet

ikemHood committed:
Fix CI… (#182)

* adjust TypeScript configuration, modify CI workflow, enhance Card component, and clean up mock tasks

* format

ikemHood commented on .env.example:
```diff
@@ -1,17 +0,0 @@
-# Since the ".env" file is gitignored, you can use the ".env.example" file to
```
Why delete file?

ikemHood reviewed (commented):
This PR should be against the develop branch not main

kelvinempires committed:
Fix package-lock.json merge conflict and reinstall dependencies

kelvinempires committed:
feat: Implement user detail and list components with loading skeletons

- Added UserDetailComponent for displaying user details and tasks.
- Created UserDetailPageWrapper to encapsulate UserDetailComponent.
- Introduced UserDetailSkeleton for loading state representation.
- Updated loading.tsx to use UserDetailSkeleton.
- Refactored page.tsx to utilize UserDetailPageWrapper with suspense.
- Developed UserListComponent for displaying a list of users with pagination and search.
- Added UserListSkeleton for loading state in user list.
- Refactored users page.tsx to use UserListComponent with suspense.
